devel/qbs/Portfile

```diff
@@ -55,7 +55,7 @@ subport ${name}-docs {
 
     universal_variant  no
     supported_archs    noarch
-    qt5.depends_build_component sqlite-plugin qttools
```

Comment:
why are these now library dependencies instead of only build dependencies? Was that a mistake earlier when using Wt5?

Reply:
Sorry, I haven't got any notifications...
It seems the syntax was changed in qt6 and qt6.depends_build_component doesn't work.
Do you have an idea how to fix this?

Reply:
@ABBAPOH I think for the `qt6` portgroup, this becomes `qt6.depends_build`

Reply:
Thanks, that works